pom.xml

```diff
@@ -37,10 +44,8 @@
     <version.assertj>3.5.2</version.assertj>
     <version.jcip.annotations>1.0</version.jcip.annotations>
     <version.shrinkwrap.resolver>2.2.5</version.shrinkwrap.resolver>
+    <version.snakeyaml>1.17</version.snakeyaml>
```

Comment:
we use it in configuration-module here: https://github.com/arquillian/arquillian-container-chameleon/pull/68/files#diff-ef6955c1e4ea3048fd0e4eaf122fa43bR54

For this classloader we should have snakeyaml dependency.It was resolving using shrinkwrap
previously (in single module). You can check [here](https://github.com/arquillian/arquillian-container-chameleon/pull/68/files#diff-ef0ec7e7b9e83bf3a6e1191127c22a38R34)